File: nice.js
function rand() {
    return Math.floor(Math.random() * 99999 + 10000)
}
let count = 0
do {
   count++
   const finder = rand()
   if(finder == 69420 || finder == 42069) {
      console.log(`${finder} Found! - ${count} times to find`)
      break
   }
   console.log(finder)
} while (true)

File: nice.lua
function rand()
    return math.random(10000, 99999)
end

local count = 0

while true do
   count = count + 1
   local finder = rand()
   if finder == 69420 or finder == 42069 then
      print(finder .. " Found! - " .. count .. " times to find!")
      break
   end
   print(finder)
end
